wgpu-types/src/lib.rs

```diff
@@ -479,6 +479,15 @@ bitflags::bitflags! {
         ///
         /// This is a native only feature.
         const SPIRV_SHADER_PASSTHROUGH = 0x0000_0800_0000_0000;
+        /// Enables `builtin(primitive_id)` in fragment shaders.
+        ///
+        /// Note: enables geometry processing, which may come with a significant performance hit on some platforms.
```

Comment:
let's mention that the hit only applies to pipelines that use this builtin semantics, it doesn't slow down everything